### PR TITLE
Upgrading -core to enable telemetry

### PR DESCRIPTION
**Pull request by rakatyal**
Related issue:
https://github.com/Microsoft/vscode-edge-debug/issues/12

### CODE REVIEW

File: src/edgeDebugSession.ts [hunk 1]
```diff
@@ -2,14 +2,14 @@
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
 
-import {ChromeDebugSession} from 'vscode-chrome-debug-core';
+import {ChromeDebugSession, IChromeDebugSessionOpts} from 'vscode-chrome-debug-core';
 import {EdgeDebugAdapter} from './edgeDebugAdapter';
 
 export class EdgeDebugSession extends ChromeDebugSession {
 
     public constructor(targetLinesStartAt1: boolean, isServer: boolean = false) {
```
Comment (roblourens) [inline src/edgeDebugSession.ts:10]: You can probably simplify things by getting rid of this class and using `ChromeDebugSession.getSession` with some args from edgeDebug.ts. See how vscode-chrome-debug does it for an example.

**Comment by roblourens** [comment]
Looks fine to me overall. You'll need someone with permissions to merge it.

File: src/edgeDebugSession.ts (hunk 1 above)
Comment (rakatyal) [reply src/edgeDebugSession.ts:10]: Is there a particular harm in doing it this way? The reason I ask is I am using the class in edgeDebugAdapter.ts as well in the constructor. I can do the reformatting if there is a good reason though.

**Comment by rakatyal** [comment]
Thanks Rob for the review!

File: src/edgeDebugSession.ts (hunk 1 above)
Comment (roblourens) [reply src/edgeDebugSession.ts:10]: Probably not, but the other way might be simpler.

**Pull request merged**